src/scheduled-query/azext_scheduled_query/tests/latest/test_scheduled_query_scenario.py

```diff
@@ -131,15 +131,15 @@ def test_monitor_scheduled_query_condition_action(self):
         self.check_condition(ns, 'Average', 'Perf | where ObjectName == \\\"Processor\\\"', 'GreaterThan', '70', '% Processor Time', 'resourceId')
 
         ns = self._build_namespace()
-        self.call_condition(ns, 'avg "% Processor Time" from "Perf | where ObjectName == \\\"Processor\\\" and C>=D && E<<F" > 70 resource id resourceId where ApiName includes GetBlob or PutBlob and DpiName excludes CCC at least 1.1 out of 10.1')
+        self.call_condition(ns, 'avg "% Processor Time" from "Perf | where ObjectName == \\\"Processor\\\" and C>=D && E<<F" > 70 resource id resourceId where ApiName includes GetBlob or PutBlob and DpiName excludes CCC at least 1.1 violations out of 10.1 aggregated points')
```

Comment:
Please correct violations and period to int in this example.

Reply:
This is not example but test. I do want to test float value. We will help customer convert the float value to int value.

Reply:
ok, thanks